Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -138,29 +137,6 @@ class ShaderCode : public ShaderGeneratorInterface
 };
 
 /**
- * Generates a shader constant profile which can be used to query which constants are used in a shader
- */
-class ShaderConstantProfile : public ShaderGeneratorInterface
-{
-public:
-	ShaderConstantProfile(int num_constants) { constant_usage.resize(num_constants); }
-
-	inline void SetConstantsUsed(unsigned int first_index, unsigned int last_index)
```

Comment:
There are still plenty "SetConstantsUsed" within the PixelShaderGen. Do you think it's worth to keep them?